portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,34 +67,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
There are only two times that post can get called-- when someone leaves a comment or when a comment is liked. In the first case, there will always be a comment and a name parameter in the req body (though either could be an empty string), hence why we check for content and name != null in the first if statement (adding a new comment). In the latter case, the key is always passed in the request body so this is how we know it is a "like". 

Since post can only be called these two times it works with the code I have written but it isn't very self explanatory or adaptable so any other suggestions are welcome :)

Reply:
Thanks for the explanation -- seems like a good case to add an inline comment with this over the if else block:)